src/modules/events/EventsCard.tsx

```diff
@@ -2,16 +2,21 @@ import React from "react";
 import EventHeader from "./EventHeader";
 import EventBody from "./EventBody";
 import Event from "./EventType";
+import { Card, CardContent, CardHeader } from "@/components/ui/card";
 
 type EventCardProps = {
   event: Event;
 };
 
 const EventCard = ({ event }: EventCardProps) => (
-  <div className="relative block transform cursor-pointer overflow-hidden rounded-lg border border-gray-100 bg-blue-100 shadow-lg transition duration-150 ease-in-out hover:scale-105">
-    <EventHeader event={event} />
-    <EventBody event={event} />
-  </div>
+  <Card className="relative block transform cursor-pointer overflow-hidden rounded-lg border border-gray-100 bg-blue-100 shadow-lg transition duration-150 ease-in-out hover:scale-105">
```

Comment:
normally, `Card` component should already have these styles as default. if not, could you please add it to the original base component, rather than applying custom styles everytime you use it?